features/factories/representative_factory.rb

```diff
@@ -39,7 +39,7 @@
     representative_mobile {'07987654321'}
     representative_reference {'Rep Ref'}
     representative_fax {'0207 345 6789'}
-    representative_contact_preference {:"simple_form.options.representative.contact_preference.post"}
+    representative_contact_preference {:"simple_form.options.claimant.contact_preference.email"}
```

Comment:
You are using the claimant contact preference key here - might confuse you one day - or it might get changed in the claimant side and break the representative side.  Up to you if you change or not